go/def.bzl

```diff
@@ -206,7 +206,8 @@ def _is_external(p):
   """Checks if the string starts with ../"""
   return p[0:3] == '../'
 
-def emit_go_compile_action(ctx, sources, deps, out_lib, extra_objects=[]):
+def _emit_go_compile_action(ctx, sources, deps, out_lib,
+                           extra_objects=[], gc_goopts=[]):
```

Comment:
I think skylark is immune to the python bug where a default argument of [] is bad, but I still see common skylark practice to follow the convention of 'None' as the default value and then a 
if not gc_goopts:
  gc_goopts = []